config/job.js

```diff
@@ -65,12 +66,13 @@ const SCHEMA_SETTINGS = Joi.object().optional();
 const SCHEMA_TEMPLATE = Joi.string().regex(Regex.FULL_TEMPLATE_NAME);
 const SCHEMA_JOB = Joi.object()
     .keys({
-        steps: SCHEMA_STEPS,
+        annotations,
```

Comment:
I think he means you'll want to have the key "annotations" followed by the actual Joi validation object for annotations